workflow/README.md

```diff
@@ -28,16 +26,62 @@ SPACEPSWD="yourpassword"
 Don't commit these changes to the repo.
 <!-- TODO: Insecure. Make this import from an environment file or the keychain. -->
 
+#### Simple case: single target
+
+Make a new configuration file with the region and time period you want to analyse. All the possible parameters are listed in [rose-suite.conf](./rose-suite.conf). You can see examples in [example](./example/). Command line usage examples are shown in [example-cylc-calls.sh](./example-cylc-calls.sh).
 
-Run the pipeline by calling:
+Run the pipeline by calling `cylc vip`, like this:
 ```bash
 cylc vip . --set-file /path/to/your/configuration/file.conf -n your-analysis-run-name
 ```
 
-Command line usage examples are shown in [example-cylc-calls.sh](./example-cylc-calls.sh).
+View progress of the pipeline by calling:
+```bash
+cylc tui
```

Comment:
At least when I run it, you need to have the name of the run when you call the tui. So something like
```cylc tui your-analysis-run-name```
would be good here.

Reply:
I'm confused: I don't have to do that at all. When I run `cylc tui`, I see all of the currently instantiated runs. I can specify which one I want to focus on using the syntax you suggest, but it's not necessary for me. What version of cylc are you running?

Reply:
I've got cylc 8.2.1:
![image](https://github.com/user-attachments/assets/be68ef01-c147-47ac-bf30-58d2caa59404)

Is there a version listed for cylc anywhere in the IFT-Pipeline anymore? I don't see a place where a cylc is listed in the yaml files--it used to be in the ift-env.yaml file.

In the cylc [docs](https://cylc.github.io/cylc-doc/stable/html/tutorial/runtime/introduction.html#the-cylc-tui) it shows that you need to provide the workflow id for `cylc tui`, but that you don't need to provide it for `cylc gui`. What version of cylc are you running?

Reply:
I'm running 8.4.0. 

In principle, the workflow should work with any version of cylc 8. I removed the ift-env.yaml at some point; all it included was `cylc-flow` and I think the instructions on the cylc website are more likely to stay up-to-date. I've included a link to those [installation instructions](https://cylc.github.io/cylc-doc/latest/html/installation.html) at the top of the file.

Could we look at what happens when you call `cylc tui` on your machine together? That might help me track down the problem.

Reply:
Added a minimum version for cylc in the README. I've been testing under cylc 8.4, so let's use that as our minimum.